package.json

```diff
@@ -2373,7 +2373,7 @@
         "serve": "webpack serve --config-name vue-hmr --mode development"
     },
     "devDependencies": {
-        "@aws-toolkits/telemetry": "1.0.21",
+        "@aws-toolkits/telemetry": "1.0.27",
```

Comment:
After https://github.com/aws/aws-toolkit-common/pull/248 (1.0.28) do we need another change in this PR?

Reply:
Yeah it's 10.0.27